src/test/emulators/storage/rules/config.spec.ts

```diff
@@ -0,0 +1,91 @@
+import * as path from "path";
+import { expect } from "chai";
+import { readFileSync } from "fs-extra";
+import { tmpdir } from "os";
+import { v4 as uuidv4 } from "uuid";
+
+import { Config } from "../../../../config";
+import { Options } from "../../../../options";
+import { RC } from "../../../../rc";
+import { getStorageRulesConfig } from "../../../../emulator/storage/rules/config";
+import { StorageRulesFiles } from "../../fixtures";
+import { Persistence } from "../../../../emulator/storage/persistence";
+import { FirebaseError } from "../../../../error";
+
+const PROJECT_ID = "test-project";
+
+describe("Storage Rules Config", () => {
+  const tmpDir = `${tmpdir()}/${uuidv4()}`;
```

Comment:
I know I did this in another PR but I think there's a more canonical way to do this. See https://github.com/firebase/firebase-tools/blob/master/scripts%2Fstorage-emulator-integration%2Ftests.ts#L112

Reply:
Done.